### PR TITLE
stanage julia docs

### PR DESCRIPTION
**Pull request by nikishe**

### CODE REVIEW

**Comment by willfurnass** [comment]
Worth doing Julia on Bessemer at the same time?

File: stanage/software/apps/julia.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+|softwarename|
+==============
+
+.. sidebar:: |softwarename|
+
+   :Versions:   |currentver|
+   :URL: https://julialang.org/
+   :Documentation:  https://docs.julialang.org/en/v1/
+   :Dependencies: EasyBuild (see :ref:`stanage_eb_toolchains`).
```
Comment (willfurnass) [inline stanage/software/apps/julia.rst:14]: Isn't it a binary install with no run-time dependencies?  Not dependent on any EB toolchains anyways.

File: stanage/software/apps/julia.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+The Julia programming language is a flexible dynamic language, appropriate for scientific and numerical computing, with performance comparable to traditional statically-typed languages. For more information visit: https://docs.julialang.org/en/v1/  
+
+Interactive Usage
+-----------------
+After connecting to Stanage,  start an interactive session with the ``srun --pty bash –i`` command.
+
+The latest version of MATLAB (currently 2022a) is made available by running:
```
Comment (willfurnass) [inline stanage/software/apps/julia.rst:22]: Julia
```suggestion
Load a particular version of Julia with:
```

File: stanage/software/apps/julia.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    println("The sum is :", result) 
+
+First, you need to write a batch submission file. We assume you’ll call this ``my_job.slurm``:   
+
+.. code-block:: bash
+
+    #!/bin/bash
+    #SBATCH --ntasks=1
+    #SBATCH --time=10:00
+    #SBATCH --mem-per-cpu=100
```
Comment (willfurnass) [inline stanage/software/apps/julia.rst:59]:
```suggestion
    #SBATCH --mem=100
```
Simpler?

File: stanage/software/apps/julia.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+Ensure that ``example.jl`` and ``my_job.slurm`` are both in your current working directory, then submit your job to the batch system:
+
+.. code-block:: bash
+    
+    sbatch my_job.slurm
+
+Installation notes
+------------------
+
+|softwarename| version 1.9.0 was installed using Easybuild 4.7.0, build details can be found
```
Comment (willfurnass) [inline stanage/software/apps/julia.rst:75]: NCDU?

Comment (willfurnass) [inline stanage/software/apps/julia.rst:75]:
```suggestion
|softwarename| version 1.9.0 was installed using Easybuild 4.8.1, build details can be found 
```

File: stanage/software/apps/julia.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+.. code-block:: bash
+    
+    sbatch my_job.slurm
+
+Installation notes
+------------------
+
+|softwarename| version 1.9.0 was installed using Easybuild 4.7.0, build details can be found 
+in ``$EBROOTNCDU/easybuild`` with the module loaded.
```
Comment (willfurnass) [inline stanage/software/apps/julia.rst:76]:
```suggestion
in ``$EBROOTJULIA/easybuild`` with the module loaded.
```

**Review by willfurnass — APPROVED** [review]
LGTM bar the need for some minor corrections.  Adding docs for Julia at Bessemer at the same time or in another PR would be good.

File: stanage/software/apps/julia.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+Interactive Usage
+-----------------
+After connecting to Stanage,  start an interactive session with the ``srun --pty bash –i`` command.
+
+Load a particular version of Julia with:
+
+.. code-block:: bash
+
+   module Julia/1.9.0-linux-x86_64
```
Comment (Carldkennedy) [inline stanage/software/apps/julia.rst:26]:
```suggestion
   module load Julia/1.9.0-linux-x86_64
```

File: stanage/software/apps/julia.rst [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+    #SBATCH --ntasks=1
+    #SBATCH --time=10:00
+    #SBATCH --mem=100
+    
+    #load the julia module
+    module load Julia/1.9.0-linux-x86_64
+
+    julia example.jl
+
+Ensure that ``example.jl`` and ``my_job.slurm`` are both in your current working directory, then submit your job to the batch system:
```
Comment (Carldkennedy) [inline stanage/software/apps/julia.rst:66]:
```suggestion
Ensure that ``example.jl`` and ``my_job.slurm`` are both in your current working directory, then submit your job to the SLURM scheduler:
```

**Review by Carldkennedy — COMMENTED** [review]
LGTM with a few suggested changes

**Pull request merged**